### PR TITLE
Ruleset static variables

### PR DESCRIPTION
**Pull request by laura-wenning**

### CODE REVIEW

File: src/components/reroll/forms/staticVariables/StaticVariableValueInput.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import { StaticVariableTextInput } from "./inputs/TextInput";
+import { StaticVariableObjectArrayInput } from "./inputs/ObjectArrayInput";
+import { StaticVariableFormValues, StaticVariableObject } from "types/components/forms/staticVariables";
+
+interface StaticVariableValueInputProps {
+  staticVariable: StaticVariable & StaticVariableFormValues;
+  setStaticVariableField: (field: string, value: any, shouldValidate?: boolean | undefined) => void;
+}
+
+export function StaticVariableValueInput(props: StaticVariableValueInputProps) {
```
Comment (laura-wenning) [inline src/components/reroll/forms/staticVariables/StaticVariableValueInput.tsx:21]: Comment

File: src/types/components/forms/staticVariables.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { StaticVariableScalarType } from "types/documents/subdocument/StaticVariable";
+
+export interface StaticVariableObject {
```
Comment (laura-wenning) [inline src/types/components/forms/staticVariables.ts:3]: Add comment definitions

**Comment by laura-wenning** [comment]
Just a heads up: the ObjectArray type is not done. I want to wrap that up in a different PR, since it's pretty complicated

**Review by lucyawrey — APPROVED** [review]
Done. I'll do more tomorrow, it's late

**Comment by laura-wenning** [comment]
Thank you <3

**Pull request merged**